### PR TITLE
Move infrastructure helpers to package

### PR DESCRIPTION
**Pull request by albertandrejev**

### CODE REVIEW

**Comment by albertandrejev** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/5726310461

**Review by swelf19 — DISMISSED** [review]

**Review by NeverHappened — APPROVED** [review]
LGTM!

File: src/testcases/parallel/subdao.test.ts [hunk 1]
```diff
@@ -126,7 +126,7 @@ describe('Neutron / Subdao', () => {
 
     test('execute timelocked: execution failed', async () => {
       //wait for timelock durations
-      await wait(20);
+      await wait.wait(20);
```
Comment (sotnikov-s) [inline src/testcases/parallel/subdao.test.ts:129]: what do you think about renaming it to waitSeconds? just to be more precise and to have it similar to the waitBlocks func

**Comment by albertandrejev** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/5726310461

**Review by sotnikov-s — DISMISSED** [review]

**Review by pr0n00gler — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -39,6 +39,7 @@
     "@cosmos-client/core": "0.45.13",
     "@cosmos-client/cosmwasm": "^0.20.1",
     "@cosmos-client/ibc": "^1.2.1",
+    "@neutron-org/neutronjsplus": "^0.0.9",
```
Comment (sotnikov-s) [inline package.json:42]: the latest version is 0.0.8, isn't it? https://github.com/neutron-org/neutronjsplus/blob/e2b4bf6482a0377b996ae8de0f66af62b5a89c6c/package.json#L3

Comment (albertandrejev) [reply package.json:42]: you are right, I forgot to push changes

**Pull request merged**